.github/workflows/pytest.yml

```diff
@@ -44,7 +45,8 @@ jobs:
       run: python -c "import sys; print(sys.version)"
     - name: Install dependencies
       run: |
-        python -m pip install --upgrade pip
+        # pin pip to workaround https://github.com/pypa/pip/issues/9180
+        python -m pip install pip==20.2
```

Comment:
```suggestion
        python -m pip install pip==20.2.4
```

Reply:
Thanks for the suggestion, but I think this is enough for now. Hopefully you'll have a patch release soon :)